esp/platform/espcontext.cpp

```diff
@@ -49,6 +49,7 @@ class CEspContext : public CInterface, implements IEspContext
     short           m_servPort;
 
     Owned<ISecUser> m_user;
+    MemoryBuffer    m_emptyMB;
```

Comment:
As m_emptyMB is not really a part of this class, it may be worth moving it outside the class (into a common shared file).